### PR TITLE
Initial commit of real code

### PR DESCRIPTION
**Pull request by jbowes**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
+
+require 'sinatra'
+require 'sinatra/json'
+require 'oauth2'
+require 'manifoldco_signature'
+
+# Manifold's public key (or your local test version), used for by the
+# manifoldco_signature gem to verifiy that requests came from Manifold.
+set :verifier_key, ENV['VERIFIER_KEY'] || ManifoldcoSignature::MASTER_KEY
```
Comment (ianlivingstone) [inline app.rb]: is there a reason this isn't just `MASTER_KEY`?

Comment (ianlivingstone) [reply app.rb]: It'd be more inline with what this key is suppose to represent, you pass the masterkey into the verifier.

**Review by ianlivingstone — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+To use Grafton to verify this sample provider:
+
+```bash
+# create a test master key for grafton to use when acting as Manifold
+# this file is written as masterkey.json
+grafton generate
+
+# Set environment variables to configure the test app
+# VERIFIER_KEY is the public_key portion of masterkey.json
+export VERIFIER_KEY="TtziSVE/9lnZ7fRYhWtGZpuXUKJ82FunK9rM0IkuP/0"
```
Comment (ianlivingstone) [inline README.md]: how can we make it more obvious that this is from the `masterkey.json` field?

File: app.rb (hunk 1 above)
Comment (jbowes) [reply app.rb]: nope! will change.

File: README.md (hunk 2 above)
Comment (jbowes) [reply README.md]: grafton generate (or grafton init, when/if we change it) should print out the values you'll need to use.

**Review by ianlivingstone — APPROVED** [review]

**Pull request merged**